notification/conf/notification.yaml

```diff
@@ -325,7 +325,7 @@ Resources:
   NotificationLaunchConfig:
     Type: AWS::AutoScaling::LaunchConfiguration
     Properties:
-      AssociatePublicIpAddress: True
+      AssociatePublicIpAddress: false
```

Comment:
I think @JamieB-gu mentioned in other PR reviews but wanted to flag here too, the titlecase on True and lowercase false

Reply:
Good point. The reason I made it lower case is that throughout all our cloudformations for boolean we are using lowercase `true` & `false` so to make it consistent I made this lowercase. 

Also, in AWS cloud formation examples I only found lowercase. Although, both lowercase and uppercase are accepted when you run update the stack, I think lowercase is the lowercase is most common :)